Makefile

```diff
@@ -1,7 +1,7 @@
 travis:
 	python setup.py test --coverage \
-		--coverage-package-name=stemmer
-	flake8 --max-complexity 10 libindic/stemmer
+		--coverage-package-name=libindic.stemmer
+	flake8 --max-complexity 15 libindic/stemmer
```

Comment:
Just to give brief on complexity, it increases as you nest too much eg. conditionals. The way to reduce it refactor code such a way you don't nest too much. I've not checked the code so I can't give you exact reason but that is the idea.

I think it was added by @arushidogra lets see if she can help or not I will take a look later.